### DISCUSSION TITLE
how to make these points in Line() disappear?

### DISCUSSION THREAD

**Discussion by Lerr1uqs** [pyecharts/pyecharts#2403] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/3838a5b7-c4e7-45d2-9aed-c83606ce25c4)

**Comment by sunhailin-Leo** [comment]
@Lerr1uqs 
* If you want a data visualization, then why are you hiding that point?

**Reply by Lerr1uqs** [reply]
> @Lerr1uqs 
> * If you want a data visualization, then why are you hiding that point?

at least I need to zoom out it, the centre point is too big

**Reply by sunhailin-Leo** [reply]
Do you mean that point style?

**Reply by sunhailin-Leo** [reply, marked as answer]
* try set https://github.com/pyecharts/pyecharts/blob/master/pyecharts/charts/basic_charts/line.py#L28 `symbol="none"`
